node/core/approval-voting/src/approval_checking.rs

```diff
@@ -74,6 +76,15 @@ impl Check {
 		match *self {
 			Check::Unapproved => false,
 			Check::Approved(_) => true,
+			Check::ApprovedOneThird => true,
+		}
+	}
+
+	/// The number of known no-shows in this computation.
+	pub fn known_no_shows(&self) -> usize {
+		match *self {
+			Check::Approved(n) => n,
+			_ => 0,
```

Comment:
```suggestion
	pub fn known_no_shows(&self) -> Option<std::num::NonZeroUsize> {
		match *self {
			Check::Approved(n) => std::num::NonZeroUsize::new(n),
			_ => None,
```

there seems to be exactly one use cases which treats `0` as special case, which really should be a `None`.

Reply:
Why NonZero?

Reply:
The option makes sense, but tbh we will always use it as unwrap_or(0), so this change would add more friction

Reply:
The 'right' change would be to expose 'covered + covering' in the other RequiredTranches variants